pages/design/atomic-template-empty.md

```diff
@@ -1,35 +1,38 @@
 ---
+id: 
 name:
 rule_type: atomic
 
 description: |
   This rule checks ...
 
-success_criterion:
-  - x.x.x # (Name of Success Criterion)
+accessibility_requirements:
+  wcag20:x.x.x: # (Name of Success Criterion)
```

Comment:
## FOR TEMPLATE UPDATE OF RULES

Today we sometimes list the conformance level next to the name of the success criterion, e.g. "# Name, Role, Value (A)". 
Do we want to do this going forward or get rid of the conformance level in all places where it is found like this?